src/app/ipv/middleware.test.js

```diff
@@ -515,6 +519,10 @@ describe("journey middleware", () => {
 
       const expectedUserDetail = {
         name: "firstName LastName",
+        nameParts: {
+          givenName: "firstName",
```

Comment:
Do you think this should test the case where you have multiple given names?

Reply:
I think it might be better to add that test case as a new test